web/app/charts/MapChart.js

```diff
@@ -22,11 +22,14 @@ bluewave.charts.MapChart = function(parent, config) {
             left: 82
         }
     };
-    var svg, mapArea; //d3 elements
+    var svg, mapArea, mapCenter; //d3 elements
     var countyData, countryData; //raw json
     var counties, states, countries; //topojson
     var options = []; //aggregation options
-
+    var projection;
+    var zoomed;
```

Comment:
I don't think we need the zoomed variable anymore so let's remove

Reply:
I have removed it from the latest version.